frontend/device/android/powerd.lua

```diff
@@ -69,6 +70,11 @@ function AndroidPowerD:turnOffFrontlightHW()
         return
     end
     android.setScreenBrightness(self.fl_min)
+
+    if device_model == "kon_tiki2" then
```

Comment:
Yes, this is correct. However, please make it:

```suggestion
    if device_model == "kon_tiki2" then -- Onyx Boox something or other
```

for clarity.

Reply:
OK! Thank you for your review :)